OutOfSchool/OutOfSchool.WebApi/Controllers/V1/RatingController.cs

```diff
@@ -57,16 +57,19 @@ public async Task<IActionResult> IsReviewed(Guid parentId, Guid workshopId)
     /// <summary>
     /// Get all ratings from the database.
     /// </summary>
+    /// <param name="skip">Skip number.</param>
+    /// <param name="take">Take number.</param>
     /// <returns>List of all ratings.</returns>
     [HasPermission(Permissions.SystemManagement)]
     [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<RatingDto>))]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-    [HttpGet]
-    public async Task<IActionResult> Get()
+    [HttpGet("{skip}/{take}")]
```

Comment:
we usually use the filter as query parameters, not path. Would be cool to keep it consistent.